inference-engine/src/preprocessing/cpu_x86_avx2/ie_preprocess_gapi_kernels_avx2.cpp

```diff
@@ -5,12 +5,19 @@
 #include <algorithm>
 #include <utility>
 
-#include "ie_preprocess_gapi_kernels.hpp"
-#include "ie_preprocess_gapi_kernels_impl.hpp"
 #include "ie_preprocess_gapi_kernels_avx2.hpp"
 
+#if defined(__GNUC__) && (__GNUC__ <= 5)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wuninitialized"
+#endif
```

Comment:
you have this in both headers and cmake. May be we can leave suppression in a single place? I would like header files

Reply:
I would prefer leave suppression in cmake because gcc-5 has one more bug. This bug leads to `#pragma GCC diagnostic ignored` don't work when lto is turned on.

Reply:
ok, agreed to keep cmake only

Reply:
Applied.